src/jsonrpc/mod.rs

```diff
@@ -174,6 +174,9 @@ impl From<commands::CommandError> for Error {
             commands::CommandError::TxBroadcast(_) => {
                 Error::new(ErrorCode::ServerError(BROADCAST_ERROR), e.to_string())
             }
+            commands::CommandError::EmptyFilterList => {
+                Error::invalid_params("Invalid 'txids' parameter.")
```

Comment:
As this is a general error type, you could make this message more generic or otherwise pass through the message from the `CommandError` itself.

Reply:
adressed